CONTRIBUTORS.md

```diff
@@ -23,6 +23,7 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
       <td align="center" valign="top" width="11.11%"><a href="https://github.com/ABostrom"><img src="https://avatars0.githubusercontent.com/u/9571933?v=4?s=100" width="100px;" alt="Aaron Bostrom"/><br /><sub><b>Aaron Bostrom</b></sub></a><br /><a href="https://github.com/sktime/sktime/commits?author=ABostrom" title="Code">💻</a> <a href="https://github.com/sktime/sktime/commits?author=ABostrom" title="Documentation">📖</a> <a href="https://github.com/sktime/sktime/commits?author=ABostrom" title="Tests">⚠️</a> <a href="#mentoring-ABostrom" title="Mentoring">🧑‍🏫</a></td>
       <td align="center" valign="top" width="11.11%"><a href="https://github.com/aaronrmm"><img src="https://avatars.githubusercontent.com/u/1742879?v=4?s=100" width="100px;" alt="Aaron Margolese-Malin"/><br /><sub><b>Aaron Margolese-Malin</b></sub></a><br /><a href="https://github.com/sktime/sktime/issues?q=author%3Aaaronrmm" title="Bug reports">🐛</a></td>
       <td align="center" valign="top" width="11.11%"><a href="https://github.com/aaronreidsmith"><img src="https://avatars.githubusercontent.com/u/21350310?v=4?s=100" width="100px;" alt="Aaron Smith"/><br /><sub><b>Aaron Smith</b></sub></a><br /><a href="https://github.com/sktime/sktime/commits?author=aaronreidsmith" title="Code">💻</a></td>
+      <td align="center" valign="top" width="11.11%"><a href="https://github.com/ShreeshaM07"><img src="https://avatars.githubusercontent.com/u/120820143?s=96&v=4" width="100px;" alt="Shreesha M"/><br /><sub><b>Shreesha M</b></sub></a><br /><a href="https://github.com/sktime/sktime/issues?q=author:ShreeshaM07" title="Bug reports">🐛</a> <a href="https://github.com/sktime/sktime/commits?author=ShreeshaM07" title="Documentation">📖</a><a href="https://github.com/sktime/sktime/commits?author=ShreeshaM07" title="Code">💻</a></td>
```

Comment:
Please do not update the CONTRIBUTORS.md instead update the .all-contributorsrc. The CONTRIBUTORS.md will be updated automatically.